### PR TITLE
BP-933  No second chance email sended on pending orders

### PR DESCRIPTION
**Pull request by saranchuk-hys**

### CODE REVIEW

File: Controller/Redirect/Process.php [hunk 1]
```diff
@@ -301,6 +301,7 @@ public function execute()
                         )
                     );
                     $this->logger->addDebug(__METHOD__ . '|5|');
+                    $this->secondChance();
                     return $this->_redirect('/');
                     // $this->redirectFailure();
```
Comment (florinel-chis) [inline Controller/Redirect/Process.php:306]: what was redirectFailure doing that it was impacting pending orders?

Comment (saranchuk-hys) [reply Controller/Redirect/Process.php]: Sorry, it was old commit, was cleared

File: Controller/Redirect/Process.php [hunk 2]
```diff
@@ -301,8 +301,8 @@ public function execute()
                         )
                     );
                     $this->logger->addDebug(__METHOD__ . '|5|');
+                    $this->secondChance();
                     return $this->_redirect('/');
```
Comment (florinel-chis) [inline Controller/Redirect/Process.php:305]: to which page is the user redirected here, after clicking on the url from the email? homepage? did we have a configurable url to be redirected to?

Comment (saranchuk-hys) [reply Controller/Redirect/Process.php:305]: yes homepage, we don`t have configurable for 791 case

File: Controller/Redirect/Process.php [hunk 3]
```diff
@@ -301,8 +301,8 @@ public function execute()
...
-                    // $this->redirectFailure();
```
Comment (florinel-chis) [inline Controller/Redirect/Process.php:305]: what was redirectFailure doing that it impacted pending orders issue?

Comment (saranchuk-hys) [reply Controller/Redirect/Process.php:305]: redirectFailure - contain configurable admin failed patch, which previews used but now redirected to homepage

**Review by florinel-chis — APPROVED** [review]

**Comment by florinel-chis** [comment]
the redirect part will be treated in a separate ticket.

**Pull request merged**